sparklightautoml/computations/manager.py

```diff
@@ -0,0 +1,448 @@
+import logging
```

Comment:
file contents could be split into more manageable pieces. the later comments will be extended later after the split